docs/manual/manual_stow.rst

```diff
@@ -0,0 +1,310 @@
+
+    Hold Stow Parameter in IQ Control Center
+
+.. _stow_movement_parameters:
+
+Stow Movement Parameters
+**************************
+The following parameters pertain to how the module will move as it transitions to its stow position:
+
+* **Stow Target Acceleration**: Sets the maximum acceleration allowed when moving to the stow position. Can be used to make the transition from spinning to the stow position more gradual.
+* **Stow Kp**: The proportional gain to use in the closed loop position controller moving the module to the stow position. A higher gain can lead to a more accurate position, but can also cause oscillation.
+* **Stow Ki**: The integral gain to use in the closed loop position controller moving the module to the stow position.
+* **Stow Kd**: The differential gain to use in the closed loop position controller moving the module to the stow position.
+
+*Stow Target Acceleration* can be found under the Advanced tab of the IQ Control Center, as shown below.
+
+.. figure:: ../_static/manual_images/stow/stow_target_acceleration_parameter.png
+    :align: center
+    :width: 60%
+    :alt: Stow Target Acceleration Parameter
+
+    Stow Target Acceleration Parameter in the IQ Control Center
+
+*Stow Kp*, *Stow Ki*, and *Stow Kd* can all be found under the Tuning tab of the IQ Control Center, as shown below.
+
+.. figure:: ../_static/manual_images/stow/stow_pid_parameters.png
+    :align: center
+    :width: 60%
+    :alt: Stow PID Parameters
+
+    Stow PID Parameters in the IQ Control Center
+
+.. _stow_position_calculation:
+
+Stow Position Calculation
+===========================
+The final stow position is calculated based on the zero angle and the target angle. The zero angle determines what angular position is considered to be zero radians. 
+The choice of this position is arbitrary, any angle can be the zero position, it is only important to be consistent so that every module is using the same reference point. 
+
+The target angle determines what angular position is considered the stow position. The target angle is always in reference to the zero angle, e.g. a target angle 
+of 1.57 radians means the stow position will be 1.57 radians counterclockwise from the zero angle. By keeping the zero angle consistent and modifying the target 
+angle between -π and π, you can set any possible stow position. The equation below summarizes this relationship between target angle, zero angle, and the final stow position.
+
+.. figure:: ../_static/manual_images/stow/stow_position_equation.png
+    :align: center
+    :width: 60%
+    :alt: Stow Position Angle Calculation
+
+    Stow Position Angle Calculation
+
+.. _stow_configuring_zero_angle:
+
+Configuring the Zero Angle
+===========================
+It is possible for users to configure the zero angle of a Vertiq module to any position. There are two approaches to changing the zero angle:
+
+#. Modify the zero angle directly, increasing or decreasing it by the desired number of radians.

```

Comment:
instead of increasing or decreasing ... how about just "setting it to any desired number of radians"